core/accounts/urls.py

```diff
@@ -6,7 +6,7 @@
 app_name = "users"
 
 urlpatterns = [
-    path("login/", auth_views.LoginView.as_view(template_name="auth-login.html"), name="login"),
+    path("login/", views.view_signin, name="login"),
```

Comment:
This is not needed. Please check how auth_views is imported, and why a custom view that handles post requests for the login is not needed.

Reply:
Consider using a custom template tag (check discord) for Django to apply the bootstrap `form-control` classes:

https://docs.djangoproject.com/en/2.2/howto/custom-template-tags/